dissect/target/plugins/os/unix/log/auth.py

```diff
@@ -279,9 +276,19 @@ def build_record(self, ts: datetime, source: Path, line: str) -> TargetRecordDes
 
         for key, value in self._parse_additional_fields(record_values["service"], line).items():
             record_type = "string"
+
             if isinstance(value, int):
                 record_type = "varint"
 
+            elif isinstance(value, list):
+                if key in ["remote_ips", "host_ips"]:
+                    record_type = "net.ipaddress[]"
+                else:
+                    record_type = "string[]"
+
+            elif key in ["remote_ip", "host_ip"]:
+                record_type = "net.ipaddress"
+
```

Comment:
```suggestion
            elif key in ["remote_ip", "host_ip"]:
                record_type = "net.ipaddress"

            if isinstance(value, list):
                record_type += "[]"

```
Or is that too broad?